cmd/root.go

```diff
@@ -96,6 +97,7 @@ func init() {
 	rootCmd.Flags().StringVarP(&cacheDir, "cache-dir", "", "", "cache dir")
 	rootCmd.Flags().StringVarP(&vulnerabilitiesTableName, "vulnerabilities-table-name", "", "vulnerabilities", "Vulnerabilities Table Name")
 	rootCmd.Flags().StringVarP(&adivisoryTableName, "advisory-table-name", "", "vulnerability_advisories", "Vulnerability Advisories Table Name")
+	rootCmd.Flags().StringArrayVarP(&sources, "sources", "", nil, "Vulnerability Sources")
```

Comment:
I like singular `--source`, because from the user's point of view, there is only one value for `--source`.

> 私は単数系のほうが好きです。ユーザからみて --source に渡す値は1だけだからです。

Reply:
👍  user friendly